roles/gpu_burn/files/gpu_burn_pod.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
```

Comment:
do you think we could use `job` instead of `pod`?

Reply:
ack, we keep the `Pod` for now, but one day we'll turn them into `Jobs`!